client.go

```diff
@@ -31,54 +27,43 @@ type Client struct {
 	ldap client.Client
 }
 
-func (c *Client) Get(conf *client.Config, dn string) (*client.Entry, error) {
-	filters := map[*client.Field][]string{
-		client.FieldRegistry.ObjectClass: {"*"},
-	}
+// UpdateDNPassword updates the password for the object with the given DN.
+func (c *Client) UpdateDNPassword(conf *client.Config, dn string, newPassword string) error {
```

Comment:
`UpdateRootPassword` was the exact same code as updating a password using a DN, so I removed it. Happy to add it back if anyone finds value in the distinction.